src/content/docs/agile-handbook/sprint-mechanics/meetings-and-ceremonies.mdx

```diff
@@ -85,70 +57,63 @@ topics:
-
-Each squad does their own backlog grooming and sprint planning, and manages their sprint backlog independently. We do retros together so we can talk through issues that affect both squads and share expertise and ideas.
-
-<Callout variant="tip">
-  Why do we end sprints on Mondays and start Tuesdays? This funny schedule makes things easier to work across time zones. If we ended sprints on Fridays, our Barcelona-based writers would need to do retros and grooming on Friday evening, and who wants that?
-</Callout>
+We break our work into one-week sprints. 
 
-## Grooming [#grooming]
+The new sprint starts on a Monday with sprint planning, where we commit to a set of stories that we're confident we can complete by the end of the sprint. On Thursday, we meet with stakeholders to get feedback on our deliverables for this sprint, and our plan for next. Then, managers and writers write new tickets for the next sprint, and a new cycle kicks off.
 
-### Best practices: [#best_practices]
+## Sprint planning [#sprint_planning]
 
-Well groomed tickets serve two purposes. First, writing a truly excellent story requires that you have a strong understanding of the scope of work and how this story will ultimately impact the user. The end user should always be first in our mind when we plan our work! Creating clear action items for every story means that before you've started any work you've already spoken to subject matter experts and stakeholders so you're already ahead of the game!
+Each Monday of a new sprint, we commit to a series of stories until we have filled our capacity for the sprint. Before the sprint planning meeting, the scrum master for each squad [calculates their point budget](/docs/agile-handbook/sprint-mechanics/planning-poker/#points_budget). 
 
-Secondly, all tickets should be groomed and ready on the Friday before sprint planning. This is because the goal is for all members of the squad to converge on the same estimation for each story which requires **everyone** to have a clear understanding of the scope of work. This means that not only do we need  good stories but we also need time and space to discuss these stories before sprint planning. 
+### Meeting rules [#meeting_rules]
 
-**The moral of the story: If we understand our own stories, then we can better explain them to our squad.** 
+* **Silent estimation:** At this point every story included in the sprint is well groomed and has been reviewed during grooming by all members of the squad. The entire squad reads each story silently. When all members of the squad have finished voting begins. 
+* **Blackout voting:**  In order to reduce influence, we blackout the cells in the spreadsheet until after a vote has happened. We change the background color to white to reveal the votes simultaneously.
```

Comment:
Might be nice to share a link to public version of the spreadsheet template if we can do that somehow. Not sure about security implications here

Reply:
I don't see why we couldn't share an empty version of this template, especially if we restrict it to New Relic

Reply:
Oh, I was thinking making it public to everyone! We know these docs get a shocking amount of traffic, and I'd be surprised if all of it was from relics